src/SDL/Video.hs

```diff
@@ -184,8 +187,9 @@ data WindowConfig = WindowConfig
   , windowOpenGL       :: Maybe OpenGLConfig -- ^ Defaults to 'Nothing'. Can not be changed after window creation.
   , windowPosition     :: WindowPosition     -- ^ Defaults to 'Wherever'.
   , windowResizable    :: Bool               -- ^ Defaults to 'False'. Whether the window can be resized by the user. It is still possible to programatically change the size by changing 'windowSize'.
-  , windowInitialSize  :: V2 CInt            -- ^ Defaults to @(800, 600)@. If you set 'windowHighDPI' flag, window size in screen coordinates may differ from the size in pixels. Use 'glGetDrawableSize' to get size in pixels.
+  , windowInitialSize  :: V2 CInt            -- ^ Defaults to @(800, 600)@. If you set 'windowHighDPI' flag, window size in screen coordinates may differ from the size in pixels. Use 'glGetDrawableSize' or 'SDL.Video.Vulkan.vkGetDrawableSize' to get size in pixels.
   , windowVisible      :: Bool               -- ^ Defaults to 'True'.
+  , windowVulkan       :: Bool               -- ^ Defaults to 'False'. If 'True', function 'SDL.Video.Vulkan.vkLoadLibrary' 'Nothing' will be called automatically before first window creation, and 'SDL.Video.Vulkan.vkUnloadLibrary' will be called after last window destruction.
```

Comment:
I also noticed this comment doesn't seem to be true - there is no call to `vkLoadLibrary` based on the value of this option.

Reply:
Sorry for the confusion, but this is just Haskell way to say what SDL does [itself](https://github.com/spurious/SDL-mirror/blob/a0acea5bcce3f47badbfeeab7bc2e36eba08fd84/include/SDL_vulkan.h#L81) :smile: